test/06_docker_images/cibuildwheel_test.py

```diff
@@ -0,0 +1,26 @@
+import subprocess, sys, os, pytest
```

Comment:
Now I'm really nitpicking, so just ignore if you want, but wouldn't the filename `test.py` be a lot shorter and less work to type?

Reply:
Projects like 02_test have a test _folder_ that contains unit tests for the package, so I wanted to have a longer name that wouldn't be confused for tests that are specific to the package.

Reply:
I hadn't thought of that indeed. I just thought `test_cibuildwheel.py` was so long, and curious why you didn't go for short